pages/spicedb/concepts/commands.mdx

```diff
@@ -28,8 +28,11 @@ The following are configuration flags that can be provided to every SpiceDB comm
 | Flag | Description | Default |
 | --- | --- | --- |
 | --datastore-conn-uri | connection string used by remote datastores (e.g. "postgres://postgres:password@localhost:5432/spicedb") |  |
+| --datastore-credentials-provider-name | retrieve datastore credentials dynamically using (%s) |  |
```

Comment:
this was copied verbatim from the code and is has a placeholder `%s`.
I can see this in other parts of this PR too

Reply:
fixed, i think?